bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,239 @@
+| s3-temperature                        | Number:Temperature         | R          | S3 Temperature Measured on PCB           | Temperature Measured on PCB |
+| s3-fault-code                         | String                     | R          | S3 FaultCode                             | 0x00 = OK. For all other values Please contact Ferroamp support |
+| s3-dc-link-voltage                    | Number:ElectricPotential   | R          | S3 DC Link Voltage                       | DC link voltage as measured by SSO-3 |
+| s3-timestamp                          | DateTime                   | R          | S3 Time Stamp When Message was Published | Time stamp when message was published |
+ |
+
+| eso-unique-identifier         | String                     | R          | Eso Unique Identifier                    | Unique identifier |
+| measured-voltage-battery      | Number:ElectricPotential   | R          | Eso Voltage Measured on Battery Side     | Measured on battery side |
+| measured-current-battery      | Number:ElectricCurrent     | R          | Eso Current Measured on Battery Side     | Measured on battery side |
+| eso-battery-energy-produced   | Number:Energy              | R          | Eso Battery Energy Produced              | Total energy produced by ESO, i.e total energy charged |
+| eso-battery-energy-consumed   | Number:Energy              | R          | Eso Battery Energy Consumed              | Total energy consumed by ESO, i.e total energy discharged |
+| eso-soc                       | Number:Dimensionless       | R          | Eso State of Charge                      | State of Charge for ESO |
+| eso-relay-status              | String                     | R          | Eso Relay Status                         | 0 = relay closed, 1 = relay open |
+| eso-temperature               | Number:Temperature         | R          | Eso Temperature Measured on PCB          | Measured inside ESO |
+| eso-fault-code                | String                     | R          | Eso FaultCode                            | See section 4.1.3.1 in Ferroamp-External-API-specifikation |
+| eso-battery-energy-produced   | Number:Energy              | R          | Eso Battery Energy Produced              | Total energy produced by ESO, i.e total energy charged |
+| eso-dc-link-voltage           | Number:ElectricPotential   | R          | Eso Dc Link Voltage                      | DC link voltage as measured by ESO |
+| eso-timestamp                 | DateTime                   | R          | Eso Time Stamp When Message was Published| Time stamp when message was published |
+
+
+| esm-unique-identifier                     | String                     | R          | Esm Unique Identifier                    | Unique identifier of battery. If available, this will be the unique id that the battery reports.|
```

Comment:
shorter = bettter, please adapt others too
```suggestion
| esm-id                     | String                     | R          | Esm Unique Identifier                    | Unique identifier of battery. If available, this will be the unique id that the battery reports.|
```

Reply:
Please, see changes